java/maven/src/org/netbeans/modules/maven/api/execute/RunConfig.java

```diff
@@ -79,6 +79,28 @@ public interface RunConfig {
 
     String getActionName();
     
+    /**
+     * Options/switches passed to maven.
+     * @return a read-only copy of the current maven options
+     * @since 2.166
+     */
+    @NonNull Map<? extends String,? extends String> getOptions();
```

Comment:
Friend API change; probably add `apichanges.xml` entry for this + bump version of the module.

Reply:
I already added entry for `options`

Reply:
Sorry, I missed it in the diff.